ut/source/test_ck_file.cpp

```diff
@@ -31,55 +32,64 @@ void test_ck_file::setup()
 
 void test_ck_file::teardown()
 {
-    remove("foo.txt");
-
-    unlink( "foo/bar" );
-    unlink("上开花/Чернобыльской");
-    rmdir("上开花");
-    rmdir("走上");
-    rmdir( "foo" );
-    rmdir( "foobar" );
+    auto scratchPath = RTF_OS_SCRATCH_DIR();
+    remove((scratchPath + ck_fs::PATH_SLASH + "foo.txt").c_str());
+
+    unlink((scratchPath + ck_fs::PATH_SLASH + "foo/bar").c_str());
```

Comment:
Just to be sure: `foo/bar` is intentional and you specifically _don't_ want `ck_fs::PATH_SLASH` here?